website/docs/releases.md

```diff
@@ -26,6 +26,15 @@ In case you are wondering why all our releases start with `0.0`, read [this FAQ
 
 :::
 
+### Upcoming Release
+
+* IANA released version 2024a of the Time Zone Database. Hyper’s time zone information is updated accordingly. Noteworthy changes:
+  * Kazakhstan unifies on UTC+5 beginning 2024-03-01.
+  * Palestine springs forward a week later after Ramadan.
+  * zic no longer pretends to support indefinite-past DST.
+  * localtime no longer mishandles Ciudad Juárez in 2422.
```

Comment:
how are those two related to Hyper?

Reply:
dono, I picked them up from the change log from IANA timezone data to 2024a. 
simply copy-pasted the summary as stated on the update instructions. 
should I remove them?